tests/virtual_machines_test.go

```diff
@@ -518,19 +518,35 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 
 			Context("and kubemacpool is opted-in on a namespace", func() {
+				var (
+					vm *kubevirtv1.VirtualMachine
+				)
```

Comment:
nit: Why 3 lines?

Reply:
no particular reason. just keeping conformity with other [tests](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/master/tests/virtual_machines_test.go#L561) that defined vm instances.
We can of course decide to change it, but if so then let's do it in another PR to also change in the other tests together.